web/themes/custom/jcc_elevated/templates/paragraphs/paragraph--content-stream.html.twig

```diff
@@ -0,0 +1,27 @@
+{{ attach_library("jcc_storybook/Accordion") }}
+{{ attach_library("jcc_storybook/Container") }}
+{{ attach_library("jcc_storybook/Stack") }}
+
+{% set classes = [
```

Comment:
I don't know what function this serves that the Body paragraph couldn't.  

If this is a new paragraph, why can't we just use the `field_body` that already exists instead of a new `field_content_stream`?

Reply:
This template was removed and will be setup later